packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionStatusInfo.tsx

```diff
@@ -9,6 +9,23 @@ import Image from 'next/image'
 import ArrowsIcon from '@/images/arrows.svg'
 
 import { useTransactionReminderInfo } from './useTransactionReminderInfo'
+import { PropsWithChildren } from 'react'
+
+export const TransactionHistoryStatusBar = ({
+  children,
+  wrapperClassName
+}: PropsWithChildren<{ wrapperClassName?: string }>) => {
```

Comment:
This is more straightforward as there's only one element in this component anyway?

```suggestion
export const TransactionHistoryStatusBar = ({
  children,
  className
}: React.HTMLAttributes<HTMLDivElement>)
```